.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+
+name: Release
+on:
+  release
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  release:
+    strategy:
+      matrix:
+        theme: [book, article]
+    name: Release ${{ matrix.theme }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Install Dependencies
+        # TODO: this only needs to install thebe install deps
+        run: npm install
+      - name: Build MyST Theme
+        run: npm run build
```

Comment:
This is annoying. We should be able to re-use the deployed artifacts. However, I don't *think* there's a trivial way to force `npm` to use the latest artifacts, and this approach avoids the possibility of a race condition.